### PR TITLE
[TBCM-22] Select Care Location Profile - Setup Api

### PR DESCRIPTION
**Pull request by leon-fw**
- GET /carelocations returns all the units in from the DB
- Tested with Postman
- Created an ICU entry to test things out

### CODE REVIEW

File: apps/api/src/unit/unit.service.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+@Injectable()
+export class UnitService {
+  constructor(
+    @InjectRepository(Unit)
+    private unitsRepository: Repository<Unit>,
+  ) {}
+
+  async getAllUnits(): Promise<Unit[]> {
+    const found = await this.unitsRepository.find();
```
Comment (karthikktamilmani-freshworks) [inline apps/api/src/unit/unit.service.ts:14]: return this.unitsRepository.find();

Comment (leon-fw) [reply apps/api/src/unit/unit.service.ts:14]: Woops, right, can simplify, thank you! 
Changed.

File: apps/api/src/unit/unit.service.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+@Injectable()
+export class UnitService {
+  constructor(
+    @InjectRepository(Unit)
+    private unitsRepository: Repository<Unit>,
+  ) {}
+
+  async getAllUnits(): Promise<Unit[]> {
+    return await this.unitsRepository.find();
```
Comment (karthikktamilmani-freshworks) [inline apps/api/src/unit/unit.service.ts:14]: This await is not needed

Comment (leon-fw) [reply apps/api/src/unit/unit.service.ts]: Thank you! Removed.

**Review by karthikktamilmani-freshworks — APPROVED** [review]

**Pull request merged**